packages/components/.storybook/clickableStyleUtils.stories.tsx

```diff
@@ -0,0 +1,208 @@
+import React from "react";
+import Button from "../src/Button";
+import { ClickableStyleProps } from "../src/ClickableStyle";
+import Heading from "../src/Heading";
+import AddRoundedIcon from "../src/Icons/AddRounded";
+import ArrowBackRoundedIcon from "../src/Icons/ArrowBackRounded";
+import ArrowForwardRoundedIcon from "../src/Icons/ArrowForwardRounded";
+import OpenInNewRoundedIcon from "../src/Icons/OpenInNewRounded";
+import Link from "../src/Link";
+import styles from "./clickableStyleUtils.stories.module.css";
+
+export const getStandardSet = (
+  Component: typeof Button | typeof Link,
+  componentName: "Button" | "Link",
+  variant: ClickableStyleProps<"button">["variant"],
+  color: ClickableStyleProps<"button">["color"] = "brand",
+) => (
+  <ul className="grid gap-y-4">
+    <li>
+      <Component color={color} variant={variant}>
+        {componentName}
+      </Component>
+    </li>
+    <li>
+      <Component color={color} variant={variant}>
+        <ArrowBackRoundedIcon
+          className={styles.arrowBackIcon}
+          purpose="decorative"
+        />{" "}
+        {componentName}
+      </Component>
+    </li>
+    <li>
+      <Component color={color} variant={variant}>
+        {componentName}
+        <ArrowForwardRoundedIcon
+          className={styles.arrowForwardIcon}
+          purpose="decorative"
+        />
+      </Component>
+    </li>
+    <li>
+      <Component color={color} variant={variant} size="medium">
```

Comment:
oop I didn't notice before -- let's include a label above the ones that are size "medium" or "small", because it's kind of subtle

Reply:
Added!